packs/data/feats.db/feathered-cloak.json

```
@@ -21,11 +21,34 @@
         "level": {
             "value": 5
         },
-        "location": "",
+        "location": null,
         "prerequisites": {
             "value": []
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "FlatModifier",
+                "predicate": {
+                    "all": [
+                        "self:disguise:non-strix"
```

```suggestion
                        "disguise:non-strix"
```
`self` isn't needed here.